streams/src/test/java/org/apache/kafka/streams/KafkaStreamsTest.java

```diff
@@ -89,6 +105,100 @@ public void before() {
         streams = new KafkaStreams(builder.build(), props);
     }
 
+    @Test
+    public void storeShouldNotFetchExpiredEvents() throws ExecutionException, InterruptedException {
```

Comment:
What's the difference of this test to the `TopologyTestDriverTest#storeShouldNotFetchExpiredEvents` below? Are they overlapping test coverage?

Reply:
I tested it using both `KafkaStreams` and `TopologyTestDriver` to be sure they both wind up with correctly wrapped stores.